pkg/chains/signing.go

```diff
@@ -44,9 +45,29 @@ import (
 const (
 	// ChainsAnnotation is the standard annotation to indicate a TR has been signed.
 	ChainsAnnotation             = "chains.tekton.dev/signed"
+	ChainsRetriesAnnotation      = "chains.tekton.dev/retries"
 	ChainsTransparencyAnnotation = "chains.tekton.dev/transparency"
+	Fail                         = "failed"
+	Pass                         = "true"
+	Retry                        = "retries"
+	MaxRetries                   = 3
 )
 
+type State int
+
+const (
+	Passed State = iota
+	Failed
+	Retrying
+	Unknown
```

Comment:
i don't think we need the Unknown state

Reply:
We need the unknown state if there isn't an annotation.